syntax = "proto3";
package alert_rule;




import "monitor_sdk/model/monitor/alert_dims.proto";
import "monitor_sdk/model/monitor/alert_conditions.proto";
import "google/protobuf/struct.proto";

/*
CreateAlertRule返回
*/
message CreateAlertRuleResponse {
message Data {
    /*
    告警规则id
    */
    string id = 1 ;
}
        
    
    /*
    code
    */
    int32 code = 1 ;
    /*
    msg
    */
    string msg = 2 ;
    /*
    data
    */
    Data data = 3 ;
}

/*
CreateAlertRuleApi返回
*/
message CreateAlertRuleResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    alert_rule.CreateAlertRuleResponse data = 4 ;
}

